Dockerfile.perses

```
@@ -7,7 +7,7 @@ COPY perses/ui ui
 
 USER 0
 
-RUN make build-ui
+RUN make build-ui NPM_INSTALL_FLAGS="--ignore-scripts"
```

@zhuje we should use the command directly as with the current version used in the sub module this will have no effect. 
```suggestion
cd ./ui && npm install --ignore-scripts && npm run build
```